### DISCUSSION TITLE
How to create your Website first on Google Search ?

### DISCUSSION THREAD

**Discussion by dhruvparmar2310** [community/community#28165] [New to GitHub]
As we all know it works on SEO but what to do in html code to make it first in every search ?
And how to use meta tag keywords and description for proper website on google search ?

### Any Suggestions :
Please reply me how to make (removed by moderator) first on Google search. What keywords I can give and how much time it takes to update in Google. 
(removed by moderator)

**Comment by tuves** [comment]
Hi @dhruvparmar2310,

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
